deploy-manage/deploy/cloud-on-k8s/elasticsearch-configuration.md

```diff
@@ -22,6 +22,7 @@ Before deploying and running ECK in production, review the basic and advanced se
 * [Node configuration](node-configuration.md): Configure the `elasticsearch.yml` of your {{es}} nodes.
 * [Volume claim templates](volume-claim-templates.md): Configure storage in your {{es}} nodes.
 * [Virtual memory](virtual-memory.md): Methods to accomplish {{es}} virtual memory system configuration requirement.
+* [Secure settings](/deploy-manage/security/secure-settings.md): Configure the {{es}} keystore through Kubernetes secrets.
```

Comment:
did you want to link directly to the k8s page here? or the parent? either is ok

Reply:
to the k8s secure settings page, great catch!